metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/DereferenceClient.java

```diff
@@ -157,4 +157,8 @@ public EnrichmentResultList dereference(String resourceId) {
     return restTemplate.exchange(dereferenceUrl, HttpMethod.GET, entity, EnrichmentResultList.class)
         .getBody();
   }
+  
+  public void setRestTemplate(RestTemplate restTemplate) {
```

Comment:
Make this method default (instead of public)? Only the test code needs to have access.

Reply:
Fixed.